### ISSUE TITLE
"502 Bad Gateway" at the page http://grafana/render

### ISSUE THREAD

**Issue by Nukles** [grafana/grafana#2603]
Hi all,

I have received a script that uses docker to create all Grafana's components on an IBM POWER8 machine (ppc64). After installation however, I get an error in the Grafana main Dashboard. A red icon appears on the top left saying "Timeseries Data Request Error" and, when clicking on it, I get the details.

![screenshot from 2015-08-26 16 12 42](https://cloud.githubusercontent.com/assets/7413240/9495736/592c015c-4c0d-11e5-86c7-92267599581d.png)

![screenshot from 2015-08-26 16 12 18](https://cloud.githubusercontent.com/assets/7413240/9495735/592b383a-4c0d-11e5-9356-3da96930362c.png)

![screenshot from 2015-08-26 16 10 58](https://cloud.githubusercontent.com/assets/7413240/9495734/592965b4-4c0d-11e5-90e9-4f6a5a4a1a6d.png)

Basically the issue is a "502 Bad Gateway" message when accessing http://grafana/render (grafana being the hostname of the Grafana machine, but in this case I use the IP Address 172.16.229.249).

In the Chrome Developer Tools this is what I get 

![screenshot from 2015-08-26 16 10 58](https://cloud.githubusercontent.com/assets/7413240/9495688/15d6b140-4c0d-11e5-87d1-8712383d8f21.png)

How can I solve this? Is the message "The specified value '!' does not conform to the required format.  The format is '#rrggbb' where rr, gg, bb are two-digit hexadecimal numbers." a clue?

Thanks!

**Comment by torkelo** [comment]
You need to ask the person who gave you the script, looks like a faulty config (and and a old version of Grafana)
